src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -42,5 +42,16 @@ public function process(ContainerBuilder $container)
         if ($container->has('fragment.handler')) {
             $container->getDefinition('twig.extension.httpkernel')->addTag('twig.extension');
         }
+
+        if ($container->has('assets.packages')) {
+            $container->getDefinition('twig.extension.assets')->addTag('twig.extension');
+        } elseif ($container->has('templating.asset.packages')) {
```

Comment:
I've done that because the new function was named `asset` as well initially. Now that the name is different, it's easier to support both. I will update the patch.